packages/iris-grid/src/GotoRow.tsx

```diff
@@ -90,7 +84,7 @@ const GotoRow = forwardRef<GotoRowElement, GotoRowProps>(
     const [isGotoRowActive, setIsGotoRowActive] = useState(false);
     let columns: dh.Column[] = [];
 
-    if (isIrisGridProxyModel(model) && model.table !== undefined) {
+    if (isIrisGridTableModelTemplate(model)) {
```

Comment:
Side note, `IrisGridModel` should just be an interface, and then the models could be classes or just objects with the correct properties or whatever.
Not for this ticket though, that'd be a bigger overhaul (as `GridModel` would also need to be an interface), and we'd need to do something about needing the model to be an EventListener (could just another type like `type UpdatingGridModel = { onUpdate: () => void }`